org.eclipse.xtext.xtext.ui.examples/projects/org.eclipse.xtext.example.homeautomation.ui/META-INF/MANIFEST.MF

```diff
@@ -1,28 +1,29 @@
-Manifest-Version: 1.0
-Bundle-ManifestVersion: 2
-Bundle-Name: Xtext Homeautomation Example - UI
-Bundle-Vendor: Eclipse Xtext
-Bundle-Version: 2.14.0.qualifier
-Bundle-SymbolicName: org.eclipse.xtext.example.homeautomation.ui; singleton:=true
-Bundle-ActivationPolicy: lazy
-Require-Bundle: org.eclipse.xtext.example.homeautomation,
- org.eclipse.xtext.example.homeautomation.ide,
- org.eclipse.xtext.ui,
- org.eclipse.xtext.ui.shared,
- org.eclipse.xtext.ui.codetemplates.ui,
- org.eclipse.ui.editors;bundle-version="3.5.0",
- org.eclipse.ui.ide;bundle-version="3.5.0",
- org.eclipse.jdt.debug.ui,
- org.eclipse.compare,
- org.eclipse.ui,
- org.eclipse.xtend.lib;resolution:=optional,
- org.eclipse.xtext.common.types.ui,
- org.eclipse.xtext.xbase.lib;bundle-version="2.14.0",
- org.eclipse.xtext.xbase.ui,
- org.eclipse.xtext.builder
-Import-Package: org.apache.log4j
-Bundle-RequiredExecutionEnvironment: JavaSE-1.8
-Export-Package: org.eclipse.xtext.example.homeautomation.ui.contentassist,
- org.eclipse.xtext.example.homeautomation.ui.quickfix,
- org.eclipse.xtext.example.homeautomation.ui.internal
-Bundle-Activator: org.eclipse.xtext.example.homeautomation.ui.internal.HomeautomationActivator
+Manifest-Version: 1.0
```

Comment:
What is the actual difference here? Please make sure line separators do not change.

Reply:
The export of the package "org.eclipse.xtext.example.homeautomation.ui.editor" is added to the manifest. I'm not sure why the diff shows the whole file as changed. When committing, the line endings should automatically be converted into unix-style, and since this file is generated it already has unix-style line endings locally.

Reply:
The line endings have automatically change to Unix style. The original file has windows line separators. So this is actually an improvement.